### DISCUSSION TITLE
📚 Add an architecture section

### DISCUSSION THREAD

**Discussion by williln** [djangopackages/djangopackages#988] [Docs]
**Is your feature request related to a problem? Please describe.**
 
Django Packages depends mainly on the relationship between Packages and Grids, if I am interpreting our conversations correctly @jefftriplett: 

So I think it would be a good idea to collect the explanation of these core concepts in its own section of the docs. Especially since the kinds of contributions you tend to receive, and seek, are contributions that deal a lot with these models. 

**Describe the solution you'd like**

Something similar to (but probably much simpler than) how Wagtail goes into their Page model: 

![Screen Shot 2023-02-10 at 3 23 40 PM](https://user-images.githubusercontent.com/2286304/218220955-82829d7b-43b5-4153-86c9-0289541656d8.png)

That's a core concept that everyone who works with Wagtail will need to deal with, so they spend extra time making sure it's clear how that works. Something similar for how Packages and Grids work would be good. 

**Describe alternatives you've considered**

None yet. 

**Additional context**

None yet.

**Comment by williln** [comment]
Data from the current Introduction page that would make a good start to an Architecture page: 

> Django Packages is a directory of Django apps, utilities, frameworks, and packages. 

Then explain that all of those would be rows in the Package table. 

> Django Packages stores information on fetched packages and provides easy comparison tools for them. 
> Public APIs include PyPI, GitHub, BitBucket, and GitLab.

This is great info for the Platforms page https://github.com/djangopackages/djangopackages/issues/911

Maybe https://github.com/djangopackages/djangopackages/issues/911 would make a good subpage for this page? 

This info on Grids would be great for an Architecture page, too. And the explanation would benefit from some examples -- a lot of the data on the site is stable enough that I think you could use live links in your docs to illustrate your point, if you wanted to. 
![Screen Shot 2023-02-10 at 3 28 08 PM](https://user-images.githubusercontent.com/2286304/218221346-18785210-4d02-4c72-a7d1-b806738b0905.png)

**Comment by jefftriplett** [comment]
@williln just noting that I moved the docs issues to discussions to free up some mental space. When I scan open issues it feels like bugs vs. improvements, but I plan to keep working on these.
